crates/cxx-qt-gen/src/naming/name.rs

```diff
@@ -97,7 +97,15 @@ impl Name {
     /// This is either:
     /// - The cxx_name attribute value, if one is provided
     /// - The original ident, if no cxx_name was provided
-    pub(super) fn cxx_unqualified(&self) -> String {
+    pub fn cxx_unqualified(&self) -> String {
         self.cxx.clone().unwrap_or_else(|| self.rust.to_string())
     }
+
+    pub fn rust_unqualified(&self) -> &Ident {
```

Comment:
docs could be useful here as to what this is

Reply:
Done